projects/vdk-core/src/vdk/api/job_input.py

```diff
@@ -151,8 +151,10 @@ def send_object_for_ingestion(
         The data is then processed and sent asynchronously by multiple background threads in parallel
          (the level of parallelism could be controlled by ingester_number_of_worker_threads configuration).
 
+        Alternatively, the same method can be used to send a pandas.DataFrame for ingestion, asynchronously.
+
         Arguments:
-            payload: dict
+            payload: Union[dict, "pandas.DataFrame"]
                 The passed object will be translated to a row in destination table.
```

Comment:
Maybe put the comment under payload. Also we should mention pandas is not installed by default. 

_The payload argument can be either a dict or a pandas.DataFrame. The pandas library is an optional dependency; ensure it is installed and included in the job's requirements before deployment_